new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -152,5 +153,15 @@ public static boolean adminT3ChannelAccessible(String domainUID, String namespac
   public static boolean adminNodePortAccessible(String domainUID, String namespace) {
     return Domain.adminNodePortAccessible(domainUID, namespace);
   }
+  
+  /**
+   * Check if a Docker image exists.
+   * @param imageName - the name of the image to be checked
+   * @param imageTag  - the tag of the image to be checked
+   * @return true if the image does exist, false otherwise
+   */
+  public static boolean assertDockerImageExists(String imageName, String imageTag) {
```

Comment:
Don't need "assert" in name.  This can just be doesDockerImageExist()

Reply:
fixed